### PR TITLE
Reacting to new Hosting API

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Reacting to https://github.com/aspnet/Hosting/pull/521

### CODE REVIEW

File: test/Microsoft.AspNet.WebSockets.Client.Test/KestrelWebSocketHelpers.cs [hunk 1]
```diff
@@ -45,12 +45,13 @@ public static IDisposable CreateServer(Func<HttpContext, Task> app)
             var config = configBuilder.Build();
             config["server.urls"] = "http://localhost:54321";
 
-            var host = new WebHostBuilder(config)
+            var appBuilder = new WebApplicationBuilder()
```
Comment (Tratcher) [inline test/Microsoft.AspNet.WebSockets.Client.Test/KestrelWebSocketHelpers.cs]: it's just called `application` elsewhere.

**Comment by Tratcher** [comment]
:shipit:

**Comment by Tratcher** [comment]
AutobahnTestServer needs updating.

**Comment by JunTaoLuo** [comment]
:up::date:

File: test/AutobahnTestServer/Project.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-    },
-    "commands": {
-        "web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://localhost:12345",
-        "kestrel": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.Kestrel --server.urls http://localhost:12344"
-    },
-    "frameworks" : {
-        "dnx451" : { },
-        "dnxcore50" : { }
-    }
+  "webroot": "wwwroot",
```
Comment (Tratcher) [inline test/AutobahnTestServer/Project.json]: Remove webroot.

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**